### ISSUE TITLE
ERROR smtrat.parser: Failed to call ... with arguments

### ISSUE THREAD

**Issue by Heizmann** [smtrat/smtrat#1]
I run [smtrat 2.1.0](https://github.com/smtrat/smtrat/files/306722/smtrat_linux64.zip) on the 
[attached SMT script](https://github.com/smtrat/smtrat/files/382684/InVarSynth_GasCakeWithJohoFractions01.bpl_Iteration1.smt2.txt) and obtained the following error message.

```
ERROR smtrat.parser: Failed to call "liipp_0__main_input" with arguments [15: liipp_0__main_output, liipp_0__main_internal, liipp_0_c, liipp_1__main_input, liipp_1__main_output, liipp_1__main_internal, liipp_1_c, liipp_2__main_input, liipp_2__main_output, liipp_2__main_internal, liipp_2_c, liipp_3__main_input, liipp_3__main_output, liipp_3__main_internal, liipp_3_c]:
        Arithmetic: Invalid operator "liipp_0__main_input".
        Bitvector: Invalid operator "liipp_0__main_input".
        Core: Arguments are expected to be formulas, but argument 1 is not: "liipp_0__main_output".
        Uninterpreted: Invalid operator "liipp_0__main_input".
```

**Comment by nafur** [comment]
Okay, this was some sort of a typo from my side. I tried to parse `(get-value <term>+)` instead of  `(get-value (<term>+))` which resulted in this error.
This should be fixed as of 37ac7e09, but it seems we introduced some problems lately with `(reset)` which we'll try to fix today.

**Comment by nafur** [comment]
Also fixed the other problem (incorrect assertion), but the problem from #2 is still remaining...
